daemon/opensnitchd.service

```diff
@@ -1,15 +1,37 @@
 [Unit]
 Description=Application firewall OpenSnitch
 Documentation=https://github.com/evilsocket/opensnitch/wiki
+Documentation=man:systemd.special
+Documentation=man:systemd.service
+Documentation=man:systemd.exec
+Documentation=man:systemd.unit
+
+DefaultDependencies=no
+Before=network-pre.target shutdown.target
+Wants=network-pre.target
+Conflicts=shutdown.target
+# Don't start when 'no-appfw` is in kernel command-line, to allow booting without it.
+ConditionKernelCommandLine=!no-appfw
```

Comment:
Is  no-appfw a well known kernel command line option, or introduced here?

Reply:
I invented it just for this service :wink: